src/map/script.cpp

```diff
@@ -28072,6 +28087,7 @@ struct script_function buildin_func[] = {
 	BUILDIN_DEF(npcstop,"??"),
 	BUILDIN_DEF(getmapxy,"rrr??"),	//by Lorky [Lupus]
 	BUILDIN_DEF(mapid2name,"i"),
+	BUILDIN_DEF(mapname2id,"i"), // By [KWDev]
```

Comment:
```suggestion
	BUILDIN_DEF(mapname2id,"i"),
```
Your authorship will be seen in the git log. Please dont take it personal.